Horion/Command/Commands/BindCommand.cpp

```diff
@@ -33,16 +30,14 @@ bool BindCommand::execute(std::vector<std::string>* args)
 			return true;
 		}
 
-		for(int i = 0; i < 190; i++)
-		{
+		for(int i = 0; i < 190; i++) {
 			const char* haystack = KeyNames[i];
 			size_t len = strlen(needle) + 1;
 			char* haystackLowercase = new char[len];
 			for (int i = 0; i < len; i++)
 				haystackLowercase[i] = tolower(haystack[i]);
 
-			if(strcmp(needle, haystackLowercase) == 0)
```

Comment:
Why not also give these variables better names?

Reply:
they're valid, see #developing channel in discord